diff --git a/py/selenium/webdriver/common/service.py b/py/selenium/webdriver/common/service.py
index 3f4299d..76ad538 100644
--- a/selenium/webdriver/common/service.py
+++ b/selenium/webdriver/common/service.py
@@ -71,6 +71,7 @@ class Service(object):
             cmd.extend(self.command_line_args())
             self.process = subprocess.Popen(cmd, env=self.env,
                                             close_fds=platform.system() != 'Windows',
+                                            stdin=PIPE,
                                             stdout=self.log_file, stderr=self.log_file)
         except TypeError:
             raise
